docs/src/app/components/pages/components/popover.jsx

```diff
@@ -87,6 +87,16 @@ let PopoverPage = React.createClass({
       <ComponentDoc
         name="Popover"
         componentInfo={componentInfo}>
+        
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nimport Popover from \'material-ui/lib/popover\';\n\n' +
```

Comment:
I can see the following issue form travis:
`ESLintError: Trailing spaces not allowed.`

> the documentation does need some heavy duplication removal

I can't agree more